src/stores/batchActions.ts

```diff
@@ -65,22 +66,12 @@ export function batchedLayout(
     try {
       await dispatch(resumeLogin(auth));
 
-      try {
-        const user = getLoggedInUser(getState());
-        if (user?.type === LoggedInUserTypes.MAPI) {
-          await dispatch(organizationsLoadMAPI(auth));
-        } else {
-          await dispatch(organizationsLoad());
-        }
-      } catch (err) {
-        const hasAccessToResources = getHasAccessToResources(getState());
-        if (!hasAccessToResources) {
-          dispatch(globalLoadFinish());
-
-          return;
-        }
-
-        throw err;
+      const user = getLoggedInUser(getState())!;
```

Comment:
This can't be `null`. `resumeLogin()` would fail if there wasn't any user available